dev/flake.nix

```diff
@@ -0,0 +1,8 @@
+{
+  description = "foo";
```

Comment:
Are these files committed by accident during manual testing? Should be removed if they are not tested programmatically.

Reply:
Oh damn it, yes!
Have been super-cautious (even while switching between branches) and then I totally forgot about them during my last force-push :facepalm: 

Sorry for that, fixed now!